website/content/en/docs/setup/installation/package-managers/helm.md

```diff
@@ -31,25 +24,14 @@ The Vector [Agent] lets you collect data from your [sources] and then deliver it
 To check available Helm chart configuration options:
 
 ```shell
-helm show values vector/vector-agent
+helm show values vector/vector
 ```
 
-This example configuration file lets you use Vector as an Agent to send logs to standard output. For more information about configuration options, see the [configuration] docs page.
+This example configuration file lets you use Vector as an Agent to send Pod logs to standard output, Vector also collects host and internal metrics and exposes them via a Prometheus exporter. For more information about configuration options, see the [configuration] docs page.
 
 ```yaml
 cat <<-'VALUES' > values.yaml
-# The Vector Kubernetes integration automatically defines a
-# kubernetes_logs source that is made available to you.
-# You do not need to define a log source.
-sinks:
-  # Adjust as necessary. By default we use the console sink
-  # to print all data. This allows you to see Vector working.
-  # /docs/reference/sinks/
-  stdout:
-    type: console
-    inputs: ["kubernetes_logs"]
-    target: "stdout"
-    encoding: "json"
+role: Agent
 VALUES
```

Comment:
This can alternatively be done at install time like: 

```shell
helm install vector vector/vector \
  --namespace vector \
  --create-namespace \
  --set role=Agent
```

Not sure what we'd prefer

Reply:
I think the `values.yaml` is fine since it leads users towards adding their own Vector configuartion.